player/local_window_player_movement.py

```diff
@@ -0,0 +1,39 @@
+import pygame
+
+from player import *
```

Comment:
It is bad habit to import with *. You should enumarate important elements.

Reply:
actually it did nothing because I still haven't use Player there, so i'll just delete this line for now